### ISSUE TITLE
Unable to see all options due to my laptop's screen ratio

### ISSUE THREAD

**Issue by wtanksleyjr** [rmcrackan/Libation#720]
**Is your feature request related to a problem? Please describe.**
My screen's aspect ratio is too long and the options don't scroll, so the options screen doesn't let me see anything below a certain point. Can't change any options at all because I can't click "Save", and can't even see any options below a certain level.

**Describe the solution you'd like**
Best option: make the CLI able to update its option files so I don't have to use my laptop at all. I use the command line except when trying to figure out new options, but since only the GUI can write to options files I can't find what I need to add to the options file.

**Describe alternatives you've considered**
Second best: a scrollbar on the options screen, or break them down so they're not as tall if that's not possible.

<img width="704" alt="Screenshot 2023-08-23 124320" src="https://github.com/rmcrackan/Libation/assets/1307641/c9cbc48c-9abf-4a7f-b39c-486f2ecd452e">

**Comment by Mbucari** [comment]
Adding scrolling to chardonnay is really easy. Classic probably isn't easy, but you're using chardonnay so that's fine.

CLI settings editor isn't going to happen, though.

**Comment by wtanksleyjr** [comment]
Yes, I should have said, Chardonnay. Thank you, it's awesome that scrolling
is easy, and that works with the way I currently work so it's all cool.

(Even though I am a bigot for CLI ;) ).

-Wm

**Comment by Mbucari** [comment]
@wtanksleyjr This has been added in [the latest release](https://github.com/rmcrackan/Libation/releases/latest).

**Comment by wtanksleyjr** [comment]
Kinda works -- I can now manually fit the settings into my screen size. The default is to have the settings the same size as they were, and I have to shrink the window by hand.

So workaround attained, I can see everything if I'm motivated enough. I'm OK with this, but maybe someday figure how to make them fit.

**Comment by wtanksleyjr** [comment]
Um... I think that comment sounded sarcastic when I said "maybe someday figure out how to make them fit." I didn't mean that sarcastically. I meant to imply I think it's just fine now, and I just wanted to note a possible future improvement.

This is a FANTASTIC program and development operation and I have nothing negative to say AT ALL.
